File: C/unit_3/Fc.c
#include <stdio.h>
int main()
{
    int yr;
    printf("Enter the Year : \n");
    scanf("%d", &yr);
    if (yr % 4 == 0)
        printf("The Year is Leap Year !!!!");
    else
        printf("The Year is Not a  Leap Year !!!!");
    return 0;
}

File: Java/unit_3/Fc.java
package Java.unit_3;

import java.util.Scanner;

public class Fc {
    public static void main(String[] args) {
        int yr;
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter the Year : \n");
        yr = sc.nextInt();
        if (yr % 4 == 0)
            System.out.println("Year is a leap Year !!!");
        else
            System.out.println("Year is  not a leap Year !!!");
        sc.close();
    }
}
